frontend/src/components/Caption.js

```diff
@@ -13,10 +15,45 @@ class Caption extends React.Component {
         super(props);
         this.state = {
             caption: props.caption, 
+            isHighlighted: false,
```

Comment:
Set `isHighlighted` from the constructor, and use it throughout instead of constantly checking `scrollTo === captionId`